test/print.cpp

```diff
@@ -168,4 +168,4 @@ TEST_F( PrintTest, print_kmap )
     }
   }
   infile.close();
```

Comment:
This change should be reverted, as we expect an empty line at the end of every file.